src/Park.js

```diff
@@ -7,22 +7,29 @@ class Park extends Component {
 
    };
  }
-  showFullCard=() =>{
+  showFullCard= () => {
 
   }
 
+  removeCard = (event) => {
+    event.preventDefault();
+    this.props.removeCard();
+  }
+
   render() {
     let imagePath = `./${this.props.selectedPark.image}`;
     return (
-      <div> 
-        <img></img>
-        <h1>{this.props.selectedPark.parkName} National Park</h1>
-        <img src={imagePath} />
-        {/* {imageSource} */}
-        <h3>{this.props.selectedPark.description.substring(0, 80)}...</h3>
-        <button onClick={this.showFullCard}>View More</button>
-        <img></img>
-        <img></img>
+      <div className="card-overlay">
```

Comment:
I like the way you laid out the image on the park popup. it does a good job of breaking up the park name and short description